### ISSUE TITLE
could you please provide CRTM 2.3.0 file ?

### ISSUE THREAD

**Issue by yefeiwu2015** [paocorrales/comGSIv3.7_EnKFv1.3#1]
Hello, I am a PhD candidate, and I am new to the GSI assimilation system. I intend to use it to assimilate satellite data to improve cloud precipitation simulation. However, I found that the download link on the original website is no longer available. I found the file comGSIv3.7_EnKFv1.3.tar for download from your side, but the installation also requires the CRTM 2.3.0 Big_Endian coefficients tarball. Could you kindly provide a copy of it? I would greatly appreciate it. My email is 2454205165@qq.com.

**Comment by paocorrales** [comment]
Hi! The CRTM coefficients are independent from GSI, which is why they are not included here. Also the size of the files. I would assume that those files are still available on the CRTM web page but it was very difficult to find anything now that they are moving to JEDI. I've found the source code and coefficients here: https://ftp.emc.ncep.noaa.gov/jcsda/CRTM/REL-2.3.0/
I hope that helps!

**Comment by yefeiwu2015** [comment]
Thank you so much for the information and the link to the CRTM 2.3.0 coefficients! This is exactly what I needed. I really appreciate your help!

Best regards!

**Comment by yefeiwu2015** [comment]
Hi, I have downloaded CRTM2.3 and noticed that there is a fix folder. Should I use the data from this folder, as shown in the image?
<img width="183" alt="fix" src="https://github.com/user-attachments/assets/37e4b5bb-b99b-492f-ad8a-fb33018f3f44">

**Comment by paocorrales** [comment]
Yes, the coefficient files are located in those folders. When running GSI you will need to link the necessary coeff files in the working directory (depending on the observations you are assimilating). I think the best place to start is checking (and using) the scripts provided with GSI. You may need to change file paths to make it work.
